source/initave.f

```diff
@@ -226,12 +226,12 @@ subroutine DecodeAverageType(AverageType)
       if (AverageType .eq. 'ADD') then
         write(*,*) 'Uncertainties treated as Additive'
         do k  = 1, NSysTot
-          SystematicForm(k) = 'ADD'
+          SysForm(k) = 11
```

Comment:
Is this intended ? String seems to be a bit more intuitive

Reply:
Now this variable stores different properties of systematic uncertainty. Namely: additive\multiplication; symmetric\up\down. Digit seems easy to parse, than string. However, indeed, less intuitive. Would you suggest to use string anyway?